docs/root/configuration/http/http_filters/router_filter.rst

```diff
@@ -257,6 +257,18 @@ in flight.
 
 The value of the header should be "true" or "false", and is ignored if invalid.
 
+x-envoy-ratelimited-reset-headers
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Setting this header will extend the list of rate limited reset headers specified in the :ref:`route configuration rate limited retry back off
+<envoy_v3_api_field_config.route.v3.RetryPolicy.RateLimitedRetryBackOff.reset_headers>`.
+
+x-envoy-ratelimited-reset-max-interval-ms
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Setting this header will override the rate limited reset max interval specified in the :ref:`route configuration rate limited retry back off
+<envoy_v3_api_field_config.route.v3.RetryPolicy.RateLimitedRetryBackOff.reset_max_interval>`.
```

Comment:
If the headers are configurable, why do we need these well defined headers? Can we remove them to make this simpler? I'm not completely clear what they do.

Reply:
This was an attempt to allow users to configure the new back off strategy using either the bootstrap or using headers, like so:

```
# bootstrap API
retry_policy:
  # as before - rename to exponential_retry_back_off ?
  retry_back_off:
    base_interval: 25
    max_interval: 250
 
  rate_limited_retry_back_off:
    reset_headers: \
      Retry-After,RateLimit-Reset,X-RateLimit-Reset,X-Rate-Limit-Reset
    reset_max_interval: 5000
 
# header API
x-envoy-ratelimited-reset-headers: \
  Retry-After,RateLimit-Reset,X-RateLimit-Reset,X-Rate-Limit-Reset
x-envoy-ratelimited-reset-max-interval-ms: 5000
```

But I'm thinking that's overkill and having the header based API is not really useful.